sysutils/apparix/Portfile

```diff
@@ -1,21 +1,22 @@
 PortSystem  1.0
 
 name            apparix
-version         08-331
+version         11-062
 categories      sysutils
 maintainers     gmail.com:matias.piipari
 platforms       darwin
+license         GPL-2+
 
 description     File system navigation on steroids
 long_description    Apparix allows fast file system navigation by bookmarking directories and \
                     jumping to a bookmark directly.
 
-homepage        http://micans.org/apparix/
+homepage        https://micans.org/apparix/
 master_sites    ${homepage}src/
 
-checksums       md5 0d7c9a2efecf5c7b44d2ce7c5844e5de \
-                sha1 0f59c16a8bdc9b9ad7ec2b75e07645cd9ef1eb9e \
-                rmd160 d79ee7a0982301872ecc11d5ac0ed2a217812e57
+checksums           rmd160  adfd2c17eafccee7025fed73d0e4d98871dd59d8 \
```

Comment:
Minor thing,  but can you clean up the checksums indenting to be in line with the rest ?

Reply:
`long_description` is also indented differently, I would fix the indentation in another commit.

Reply:
This PR is the one changing the indentation for the checksums, so that one at least should be fixed here, before this is merged. `long_description` can indeed be fixed later, but checksums should be put back before this is merged.

Reply:
Indentation fixed.